code/src/components/TaskItems.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import { useDispatch } from 'react-redux'
+import { tasks } from 'reducers/Tasks'
+
+import moment from "moment"
+
+import './TaskItems.css'
+
+export const TaskItems = (props) => {
```

Comment:
No need for parenthesis if you only pass one argument (prettier will fix this for you automatically 👍(ironisk))